src/styles/common/_buttons.scss

```diff
@@ -56,7 +56,7 @@
 
         .bdl-btn-icon {
             // Icon without text:
-            margin: ($bdl-btn-height - $bdl-btn-icon-size) / 2 - $bdl-btn-border-width;
+            margin: ($bdl-btn-height - $bdl-btn-icon-size) * .5 - $bdl-btn-border-width;
```

Comment:
Used Sass-Migrator to auto replaced all divisions. For the case "/ 2", it will change it to "* .5". Otherwise, it will change to math.div(*, *).

Ref: https://sass-lang.com/documentation/breaking-changes/slash-div